### PR TITLE
fluid rows

### PR DESCRIPTION
**Pull request by agitator**

### CODE REVIEW

File: src/plone/app/mosaic/profiles/default/registry.xml [hunk 1]
```diff
@@ -233,6 +234,17 @@
     <value key="weight">220</value>
   </records>
 
+  <records interface="plone.app.mosaic.interfaces.IFormat"
```
Comment (jensens) [inline src/plone/app/mosaic/profiles/default/registry.xml]: After adding records an upgrade step loading the new record into existing installations is necessary.

**Comment by datakurre** [comment]
Thanks @jensens.

File: src/plone/app/mosaic/profiles/default/registry.xml (hunk 1 above)
Comment (agitator) [reply src/plone/app/mosaic/profiles/default/registry.xml]: added upgrade step

**Comment by agitator** [comment]
fluid rows now also in edit mode and works with subcolumns

**Comment by agitator** [comment]
@datakurre merge and time for a release?

**Comment by datakurre** [comment]
I don’t know what version to release. (And I’m still unable to touch Mosaic for some time.) I can share PyPI permissions... :) 

2.0 has been branched, but blocking issues in issuetracker remain unfixed.

Master is for 2.1, so should I release 2.1a1 before 2.0 is out?

**Comment by jensens** [comment]
anyway, we can merge this one. release is then another thing.

I do not have a problem releasing a 2.1a1 before 2.0. final is out. but other may feel confused.

**Comment by agitator** [comment]
+1 for 2.1a1

**Pull request merged**